code/client/clrcore/External/VehicleHash.cs

```diff
@@ -1,481 +1,526 @@
-namespace CitizenFX.Core
+﻿namespace CitizenFX.Core
 {
-	public enum VehicleHash : uint
-	{
```

Comment:
Should be fixed in https://github.com/mraes/fivem/commit/9a567ffbae1643197d8c044efe957ddd68ec4a4e. You'll also see that some hashes changed because they were just wrong.
Largely copied from https://github.com/crosire/scripthookvdotnet/commit/8e9103a7128b9a9290dddadff10ddca3822fc16d